pallets/pips/src/lib.rs

```diff
@@ -112,9 +113,13 @@ use polymesh_primitives_derive::VecU8StrongTyped;
 #[cfg(feature = "std")]
 use serde::{Deserialize, Serialize};
 use sp_core::H256;
-use sp_runtime::traits::{BlakeTwo256, CheckedAdd, Dispatchable, Hash, Saturating, Zero};
+use sp_runtime::traits::{
+    BlakeTwo256, CheckedAdd, CheckedSub, Dispatchable, Hash, Saturating, Zero,
+};
 use sp_std::{convert::From, prelude::*};
 
+const PIPS_LOCK_ID: LockIdentifier = *b"pips    ";
```

Comment:
is there a specific reason for extra spaces?

Reply:
Yes, `LockIdentifier` is [u8; 8] and the extra spaces fill out the length or otherwise we have `[u8; 4]` and a type error.